API.md

```diff
@@ -503,36 +752,119 @@ frisby.create('First test')
 ```
 
 ### only()
+
 Exclusively run this test. When `toss()` is invoked, the test is wrapped in a
 Mocha `describe.only` block instead of a `describe` block.
 
+### timeout(ms)
+
+Sets the timeout for this request.
+
+- Types: `ms`: `string`
+- Defaults: `none`
+
+```javascript
+frisby.create('Long-running request')
+    .get('http://example.com/slowthing')
+    .timeout(5000)
+    .expectStatus(200)
+    .toss()
+```
+
+This function can also be called with no parameter to return the current configured timeout (either by default, by global setup or having used this function with a parameter previously).
+
+### retry(count, backoff)
+
+Set the number of (and additional backoff between) retries for this test. Each retry will be the configured timeout plus (retry number x backoff) apart.
+
+- Types: `count`: `integer`, `backoff`: `integer`
+- Defaults: `backoff`: 1000
+
+```javascript
+frisby.create('Get a flaky thing')
+    .get('http://example.com/thing-that-sometimes-responds')
+    .expectStatus(200)
+    .retry(2, 250)
+    .toss()
+```
+
+### baseUri(uri)
+
+Set the root URI/URL that will be prepended to every request, replacing anything set by `request.baseUri` in global setup.
+
+```javascript
+frisby.create('Simple Get')
+  .baseUri('http://httpbin.org')
+  .get('/get?foo=bar')
+  .expectStatus(200)
+.toss()
+```
+
+### waits(ms)
+
+Sets a period of time in milliseconds to wait after `toss()` before the request is sent
+
+```javascript
+const myUser = {name: 'Jane Doe'}
+frisby.create('Create Item')
+    .post('http://example.com/users/create', myUser)
+    .after(function(err, res, body, headers) {
+        frisby.create('Check Item Exists')
+            .get('http://example.com/users/list')
+            .waits(500) //Allow time for extra server-side processing of the user
```

Comment:
This comment could be moved to the method comment:

> Sets a period of time in milliseconds to wait after `toss()` before the request is sent. This can allow time for server-side processing between chained requests.

Reply:
🤦‍♂️  Yeah. Fixed.